schemas.go

```diff
@@ -38,8 +38,10 @@ func (p *ProviderSchemas) Validate() error {
 		return errors.New("unexpected provider schema data, format version is missing")
 	}
 
-	if ProviderSchemasFormatVersion != p.FormatVersion {
-		return fmt.Errorf("unsupported provider schema data format version: expected %q, got %q", PlanFormatVersion, p.FormatVersion)
+	oldVersion := "0.1"
```

Comment:
_Minor_ assuming that this is going to be logic that will need to be changed in the future.

Reply:
Yes, I assume so, but from what I understand it most likely won't happen until the next major version of Terraform itself.